apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue

```diff
@@ -53,21 +52,21 @@ const state = reactive({
         key: 'budget_usage',
         desc: true,
     } as Query['sort'],
-    budgetUsageParams: computed(() => {
-        budgetUsageApiQueryHelper
-            .setFilters(state.queryStoreFilters)
-            .setPage(state.pageStart, state.pageLimit);
-        const _query = getBudgetUsageAnalyzeRequestQuery(state.sort, state.period);
-        return {
-            query: { ..._query, ...budgetUsageApiQueryHelper.data },
-        };
-    }),
 });
 
 const fetchBudgetUsages = async (): Promise<{more: boolean; results: BudgetUsageAnalyzeResult[]}> => {
     try {
         state.loading = true;
-        return await SpaceConnector.clientV2.costAnalysis.budgetUsage.analyze(state.budgetUsageParams);
+        budgetUsageApiQueryHelper
+            .setFilters(state.queryStoreFilters)
+            .setPage(state.pageStart, state.pageLimit);
```

Comment:
question: Why is this moved into here instead of computed hook? Isn't it inefficient because of recalculation on every list api?

Reply:
I'll move it back to computed 😅